node/impl/full/state.go

```diff
@@ -810,3 +811,35 @@ func (a *StateAPI) StateMinerAvailableBalance(ctx context.Context, maddr address
 
 	return types.BigAdd(st.GetAvailableBalance(act.Balance), vested), nil
 }
+
+// StateVerifiedClientStatus returns the data cap for the given address.
+// Returns ErrNotFound if there is not entry in the data cap table for the
+// address.
+func (a *StateAPI) StateVerifiedClientStatus(ctx context.Context, addr address.Address, tsk types.TipSetKey) (verifreg.DataCap, error) {
+	act, err := a.StateGetActor(ctx, builtin.VerifiedRegistryActorAddr, tsk)
+	if err != nil {
+		return verifreg.DataCap{}, err
+	}
+
+	cst := cbor.NewCborStore(a.StateManager.ChainStore().Blockstore())
+
+	var st verifreg.State
+	if err := cst.Get(ctx, act.Head, &st); err != nil {
+		return verifreg.DataCap{}, err
+	}
+
+	vh, err := hamt.LoadNode(ctx, cst, st.VerifiedClients)
+	if err != nil {
+		return verifreg.DataCap{}, err
+	}
+
+	var dcap verifreg.DataCap
+	if err := vh.Find(ctx, string(addr.Bytes()), &dcap); err != nil {
+		if err == hamt.ErrNotFound {
+			return verifreg.DataCap{}, api.NotFoundErr
```

Comment:
You can't compare errors when the method is invoked over JsonRPC. Instead this method could be returning the data cap as a pointer, with nil/null indicating 'not found'